### DISCUSSION TITLE
Why not use connection pools in budibase?

### DISCUSSION THREAD

**Discussion by dhfyes** [Budibase/budibase#6127] [Help]
Why not use connection pools in budibase instead of other methods? 
When MySQL database queries are called concurrently, the more SQL is executed at the same time, It will be very slow.
Is there any environment variable that can be set to the budibase use connection pool？
```
async connect() {
      this.client = await mysql.createConnection(this.config)
    }
```

https://github.com/sidorares/node-mysql2#using-connection-pools

**Comment by PClmnt** [comment]
Hi @dhfyes

We had to turn this off due to issues with connection pooling in our cloud envrionment, due to the nature of the cloud setup, it just wasn't performant.

However, you are right that we should let self-hosting users enable this, I have created an issue to track this: [#6310](https://github.com/Budibase/budibase/issues/6130)

**Reply by dhfyes** [reply]
Ok, thank you very much!
